libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -983,6 +1025,10 @@ static DNNReturnType execute_model_tf(TFRequestItem *request, Queue *inference_q
     TaskItem *task;
 
     inference = ff_queue_peek_front(inference_queue);
+    if (!inference) {
+        av_log(NULL, AV_LOG_ERROR, "Failed to get inference item\n");
+        return DNN_ERROR;
+    }
     task = inference->task;
```

Comment:
basically ok for patch: lavfi/dnn_backend_tf: Error Handling.

You may send the first 6 patches to mail list. 

It is async functions starting from the next patch, and I'll find other time for the review.

Reply:
Okay. I'll send them once I add the deletion of output_tensors in `tf_free_request`.

Thank you for reviewing.

Reply:
well done, thanks.